python/ray/rllib/tests/test_supported_spaces.py

```diff
@@ -17,6 +17,7 @@
 ACTION_SPACES_TO_TEST = {
     "discrete": Discrete(5),
     "vector": Box(-1.0, 1.0, (5, ), dtype=np.float32),
+    "multidiscrete": MultiDiscrete([1, 2, 3, 4]),
```

Comment:
Should there be an explicit test for the case that failed earlier? It's possible that this test already covers it though.

Reply:
Yep, this covers the failing case. The test fails before the other changes in the PR.